stock-usecase/src/main/java/org/dhis2/android/rtsm/services/StockTableDimensionStore.kt

```diff
@@ -0,0 +1,165 @@
+package org.dhis2.android.rtsm.services
+
+import javax.inject.Inject
+import org.dhis2.composetable.model.TableModel
+import org.hisp.dhis.android.core.D2
+
+class StockTableDimensionStore @Inject constructor(
+    val d2: D2
+) {
+    private var programUid: String = ""
+    private var sectionUid: String = ""
```

Comment:
why not `: String?` instead of empty string?.

Reply:
I need them to be no null as they are used for building the keystore key value